src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Proxy/SessionHandlerProxyTest.php

```diff
@@ -127,7 +127,7 @@ public function testGc()
      */
     public function testValidateId()
     {
-        $mock = $this->getMockBuilder(['SessionHandlerInterface', 'SessionUpdateTimestampHandlerInterface'])->getMock();
+        $mock = $this->getMockBuilder(TestSessionHandler::class)->getMock();
```

Comment:
💡 PHPUnit 9 does not allow to mock multiple interfaces at once anymore. That's why I've created a dummy class that is mocked instead.

Reply:
@derrabus actually since the version 8.5 PHPUnit we are not able to mock multiple interfaces at once.